tests/rptest/tests/cluster_health_overview_test.py

```diff
@@ -71,9 +77,13 @@ def is_healthy():
 
         wait_until(is_healthy, 30, 2)
 
+    def split_ntp(self, ntp: str):
+
```

Comment:
Excessive blank line.

Also this function is not in use. I'd actually separate extraction of the topic name into the function as well so that we don't need to take the item no. 1 of the triplet every time we use it.